README.md

```diff
@@ -50,14 +48,39 @@ store.add(certificate_object)
 store.close
 ```
 
+### Get certificate
+
+This method get certificate object from open certificate store.
```

Comment:
Gets a certificate from an open certificate store and returns it as an `OpenSSL::X509` object.